package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "",
-  "version": "",
+  "name": "Reflective",
+  "version": "1.0.0-beta",
   "description": "",
```

Comment:
Virtual Journal that calls you once a day for 60s, converts your call to text entries and visualizes insights on the sentiment, emotions and keywords.